docs/content/integrations/airlift.mdx

```diff
@@ -1 +1,45 @@
-This is the main page.
+# Airlift
+
+Airlift is a toolkit for integrating Dagster and Airflow.
+
+- Observe Airflow instances from within Dagster
+- Accelerate the migration of Airflow DAGs to Dagster assets with opinionated tooling.
+
+---
+
+## Compatibility
+
+### REST API Availability
+
+Airlift depends on the the availability of Airflow’s REST API. Airflow’s REST API was made stable in its 2.0 release (Dec 2020) and was introduced experimentally in 1.10 in August 2018. Currently Airflow requires the availability of the REST API.
+
+- **OSS:** Stable as of 2.00
+- **MWAA**
+  - Note: only available in Airflow 2.4.3 or later on MWAA.
+- **Cloud Composer:** No limitations as far as we know.
+- **Astronomer:** No limitations as far as we know.
+
+---
+
+## Airflow Migration Tutorial
+
+In this tutorial, we'll use `dagster-airlift` to migrate an Airflow dag to Dagster asssets. [Click here to get started](/integrations/airlift/tutorial).
```

Comment:
Can "Click here to get started" be moved before the following bullet points?

That way people read the full content before navigating away?

Reply:
Yeah! whoops.